egs2/TEMPLATE/asr1/utils/utils

```diff
@@ -0,0 +1 @@
+utils/
```

Comment:
Is this used somewhere?

Reply:
No, this was a mistake. I have fixed it.